.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+ratings:
+  paths:
+  - "**.js"
+  - "**.jsx"
+exclude_paths:
+- dist/
+- test/
```

Comment:
@whabib I don't think we should ignore `test/` because it contains the `support` tools and our specs should be held to the same standards as the rest of our code.